### ISSUE TITLE
[BUG]: Incomplete Airport dropdown

### ISSUE THREAD

**Issue by FormicAcidGD** [Zedruc/ATC24-Suite-Feedback#24]
Some airports that are able to be controlled on atc24 are not showing in the aircraft dropdown.
This is (non-exhaustive): Garry (IGAR), Boltic (IBLT), Henstridge (IHEN), Mcconnell (IIAB), Scampton (ISCM), Skopelos (ISKP), Training Centre (ITRC), Tavaro (TVO), Sea Haven (SHV), Waterloo (OWO)

**Comment by Zedruc** [comment]
Yeah those are controlled twice a year and were deliberately left out so I have less work 🫡
